stdlib/REPL/src/REPL.jl

```diff
@@ -451,7 +447,11 @@ mutable struct LineEditREPL <: AbstractREPL
         new(t,hascolor,prompt_color,input_color,answer_color,shell_color,help_color,history_file,in_shell,
             in_help,envcolors,false,nothing, Options(), nothing, Tuple{String,Int}[])
 end
-outstream(r::LineEditREPL) = r.t
+function outstream(r::LineEditREPL)
+    linfos = r.last_shown_line_infos
+    empty!(linfos)
+    IOContext(r.t, :LAST_SHOWN_LINE_INFOS => linfos)
+end
```

Comment:
```suggestion
outstream(r::LineEditREPL) = r.t
```

Reply:
Interestingly enough, that doesn't work for `show(::MethodTable)` anymore. Will have to investigate.